src/extension/app-schema/app-schema-test/src/test/resources/test-data/Observation.json

```diff
@@ -0,0 +1,332 @@
200.0]]], GeometryAttributeImpl:shape<GeometryPropertyType crs=GEOGCS[\"WGS 84\", \n  DATUM[\"World Geodetic System 1984\", \n    SPHEROID[\"WGS 84\", 6378137.0, 298.257223563, AUTHORITY[\"EPSG\",\"7030\"]], \n    AUTHORITY[\"EPSG\",\"6326\"]], \n  PRIMEM[\"Greenwich\", 0.0, AUTHORITY[\"EPSG\",\"8901\"]], \n  UNIT[\"degree\", 0.017453292519943295], \n  AXIS[\"Geodetic latitude\", NORTH], \n  AXIS[\"Geodetic longitude\", EAST], \n  AUTHORITY[\"EPSG\",\"4326\"]]>=POLYGON ((52.5 -1.2, 52.6 -1.2, 52.6 -1.1, 52.5 -1.1, 52.5 -1.2)), ComplexAttributeImpl:samplingFrame<SpatiallyExtensiveSamplingFeaturePropertyType>=[], ComplexAttributeImpl:specification<GeologicFeaturePropertyType>=[FeatureImpl:GeologicUnit<GeologicUnitType id=gu.25699>=[ComplexAttributeImpl:description<StringOrRefType>=[AttributeImpl:simpleContent<string>=Olivine basalt, tuff, microgabbro, minor sedimentary rocks], ComplexAttributeImpl:name<CodeType>=[AttributeImpl:simpleContent<string>=Yaugher Volcanic Group], ComplexAttributeImpl:name<CodeType>=[AttributeImpl:simpleContent<string>=-Py], ComplexAttributeImpl:observationMethod<CGI_TermValuePropertyType>=[ComplexAttributeImpl:CGI_TermValue<CGI_TermValueType>=[ComplexAttributeImpl:value<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=urn:ogc:def:nil:OGC::missing]]], AttributeImpl:purpose<DescriptionPurposeType>=instance, ComplexAttributeImpl:geologicUnitType<ReferenceType>=[], ComplexAttributeImpl:composition<CompositionPartPropertyType>=[FeatureImpl:CompositionPart<CompositionPartType id=cp.167775491936278899>=[ComplexAttributeImpl:role<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=fictitious component], ComplexAttributeImpl:proportion<CGI_ValuePropertyType>=[FeatureImpl:CGI_TermValue<CGI_TermValueType id=nonexistent>=[ComplexAttributeImpl:value<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=nonexistent]]], ComplexAttributeImpl:lithology<ControlledConceptPropertyType>=[FeatureImpl:ControlledConcept<ControlledConceptType id=cc.5>=[ComplexAttributeImpl:name<CodeType>=[AttributeImpl:simpleContent<string>=name_cc_5], ComplexAttributeImpl:name<LocalizedGenericNameType>=[AttributeImpl:simpleContent<string>=name_cc_5], ComplexAttributeImpl:vocabulary<GeologicVocabularyPropertyType>=[]]]]], ComplexAttributeImpl:exposureColor<CGI_TermValuePropertyType>=[FeatureImpl:CGI_TermValue<CGI_TermValueType id=Blue>=[ComplexAttributeImpl:value<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=Blue]]], ComplexAttributeImpl:outcropCharacter<CGI_TermValuePropertyType>=[FeatureImpl:CGI_TermValue<CGI_TermValueType id=x>=[ComplexAttributeImpl:value<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=x]]], ComplexAttributeImpl:occurrence<MappedFeaturePropertyType>=[]]], ComplexAttributeImpl:observationMethod<CGI_TermValuePropertyType>=[ComplexAttributeImpl:CGI_TermValue<CGI_TermValueType>=[ComplexAttributeImpl:value<CodeWithAuthorityType>=[AttributeImpl:simpleContent<string>=urn:ogc:def:nil:OGC::missing]]]]"
```

Comment:
@nmco there are four instances of ```result``` in ```Observation.json``` that are being encoded as the Java ```toString``` of the nested feature. This must be wrong.